pkg/collectors/cluster_resource_quota.go

```diff
@@ -74,6 +75,77 @@ var (
 					m.LabelKeys = []string{"resource", "type"}
 				}
 
+				return f
+			}),
+		},
+		metric.FamilyGenerator{
+			Name: "openshift_clusterresourcequota_namespace_usage",
+			Type: metric.MetricTypeGauge,
+			Help: "Usage about clusterresource quota per namespace.",
+			GenerateFunc: wrapClusterResourceQuotaFunc(func(r *v1.ClusterResourceQuota) metric.Family {
+				f := metric.Family{}
+
+				for _, rq := range r.Status.Namespaces {
+					for res, qty := range rq.Status.Hard {
+						f.Metrics = append(f.Metrics, &metric.Metric{
+							LabelValues: []string{string(rq.Namespace), string(res), "hard"},
+							Value:       float64(qty.MilliValue()) / 1000,
+						})
+					}
+					for res, qty := range rq.Status.Used {
+						f.Metrics = append(f.Metrics, &metric.Metric{
+							LabelValues: []string{string(rq.Namespace), string(res), "used"},
+							Value:       float64(qty.MilliValue()) / 1000,
+						})
+					}
+
+					for _, m := range f.Metrics {
+						m.LabelKeys = []string{"namespace", "resource", "type"}
+					}
+				}
+
+				return f
+			}),
+		},
+		metric.FamilyGenerator{
+			Name: "openshift_clusterresourcequota_selector",
+			Type: metric.MetricTypeGauge,
+			Help: "Selector of clusterresource quota, which defines the affected namespaces.",
+			GenerateFunc: wrapClusterResourceQuotaFunc(func(r *v1.ClusterResourceQuota) metric.Family {
+				f := metric.Family{}
+
+				sel := r.Spec.Selector
+				labelKeys := []string{"type", "key", "values"}
+				for key, value := range sel.AnnotationSelector {
+					f.Metrics = append(f.Metrics, &metric.Metric{
+						LabelKeys:   labelKeys,
+						LabelValues: []string{"annotation", string(key), string(value)},
+						Value:       float64(1),
+					})
+				}
+
+				if sel.LabelSelector != nil {
+					labelKeys := []string{"type", "operator", "key", "values"}
+
+					for key, value := range sel.LabelSelector.MatchLabels {
+						f.Metrics = append(f.Metrics, &metric.Metric{
+							LabelKeys:   labelKeys,
+							LabelValues: []string{"label", "In", string(key), string(value)},
```

Comment:
hardcode the operator "In" here is not correct,  there is no operator for `MatchLabels`, I'd suggest:
1. For `MatchLabels`
    {type="match-labels", key="key", value="value"}
2. For `MatchExpressions`
   { type="match-expressions, key="key", operator="op", value="value" }

Reply:
ok.